components/BannerGraphic.vue

```diff
@@ -27,7 +27,7 @@ export default {
 <template>
   <div v-if="shown" class="banner-graphic" :class="{'small': small}">
     <div class="graphic">
-      <BrandImage class="banner" file-name="banner.svg" :draggable="false" />
+      <BrandImage class="banner" file-name="banner.svg" :draggable="false" alt="Banner Image" />
```

Comment:
I think if we're doing this properly, all of these need to be localised rather than using English

Reply:
translations added, please review